addusers.py

```diff
@@ -137,12 +137,15 @@ def create_user(self, fullname, username, password, description, email, project_
             
             setpass_token = self.setpass.get_token(user.id, password, pin)
             password_url = self.setpass.get_url(setpass_token)
-
-            usr_cfg = dict(config.items('welcome_email'))
-            pwd_cfg = dict(config.items('password_email'))
-
+        
+            usr_cfg = email_defaults.copy() 
```

Comment:
NOTE: email_defaults is loaded in `__main__` (see line 275) and not explicitly passed down into this function.  It's probably not the best way to handle this - suggestions?

I could:

* pass it explicitly as an arg into the openstack function(s) that send email
* make it an attribute of the OpenStack class
* load the values separately here and down at the listserv function (line 312)

Worth thinking about this now because future improvements to this script may involve additional kinds of emails (for example, an email to an existing user who requested a new project.)